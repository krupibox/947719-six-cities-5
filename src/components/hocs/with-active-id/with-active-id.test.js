import withActiveId from './with-active-id';

describe(`<withActiveCoords/>: `, () => {

  const MockComponent = () => <div />;
  const MockWrapper = withActiveId(MockComponent);

  it(`should render correctly`, () => {
    const wrapper = renderer
      .create(
          <MockWrapper/>
      ).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
