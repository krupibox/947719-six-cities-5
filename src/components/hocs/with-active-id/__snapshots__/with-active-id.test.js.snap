// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<withActiveCoords/>:  should render correctly 1`] = `<div />`;
